### DISCUSSION TITLE
利用AI來幫忙工作

### DISCUSSION THREAD

**Discussion by 11timyyy** [1101-GS4719-C/Python-Community-GS4719-C#542] [General]
美國最近的調查發現，有著七成的美國消費者希望工作有 AI 協助，他們希望AI能幫忙除錯、解決問題、簡化流程等等。
更多如下圖:
![image](https://user-images.githubusercontent.com/100007643/161250306-397816c2-c0ad-4f35-926b-2338231ef75f.png)

而當然，AI是扮演著「幫忙者」的腳色，而非做決策者，在利用AI的同時，也要注意不可被其取代!
有些人則不喜歡AI幫忙做工，主要的考量為隱私以及安全問題。
無論如何，人們對於新科技都要小心地使用，不要反而被控制了。

文章來源:https://technews.tw/2022/03/31/70-of-u-s-consumers-want-to-use-ai-for-their-jobs/

**Comment by JunTingLin** [comment]
AI除了可以幫忙工作外，竟然還可以用AI寫程式，可以參考[😲這太強啦!AI幫你寫程式(最牛插件).md #544](https://github.com/NCU-GS4719-Python/Python-Community/discussions/544)

**Reply by liangyinglly** [reply]
我寫Java的時候使用機器人 真的超級快

**Comment by leon10456** [comment]
感覺之後會是主流

**Comment by ThomasMeng0515** [comment]
AI確實可以幫我們作許多繁瑣花時間的事，但也可能造成某些人失業，所以在兩者之間取得平衡很重要

**Comment by hapiwangy** [comment]
科技的發展本身就是帶給人們便利，期許自己不要因為科技反而使人類的其他能力退化

**Comment by ghost** [comment]
我是覺得還不用太擔心人類會被AI完全取代，提升自己的價值比較實際就是惹

**Comment by Jay-722** [comment]
AI拿來幫助工作可以使的繁瑣的工作變得更有效率，且只是擔任協助者的角色。

**Comment by SamTU0222** [comment]
這個未來一定是主流 不過如何取得平衡還是一大問題

**Comment by sean1325418** [comment]
AI就是適合拿來做重複性很高的動作，感覺以後工廠的工作都會被AI+機械手臂取代

**Comment by bloodtrail0511** [comment]
學分學程的另一門課AI導論也介紹了AI與工、商業之間的應用與商業價值

**Comment by cordydeng** [comment]
利用AI可以幫助我們更快的完成許多事情 , 而且隨著科技的進步 , AI也有極大的可能會完全取代部分工作

**Comment by PoooYen** [comment]
處理繁複的工作AI確實是非常好的幫手

**Comment by Sazmafuji** [comment]
AI真的是萬能

**Comment by HuanYuuuuuu** [comment]
利用AI來處理簡單又繁瑣的工作的確是很不錯

**Comment by d26488399** [comment]
感覺很棒，可以提高工作效率

**Comment by Yuhsin000** [comment]
Double check 感覺會比較安心

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望人類與AI可以找到平衡。

**Comment by minniewei** [comment]
感覺在未來我們的生活周遭都會充斥者ai

**Comment by henrywen547** [comment]
AI 能幫助人類更快更精準地解決問題，但也要思考如何在人力被取代以及AI是人類的助手之間做出達到平衡

**Comment by Rosancl** [comment]
隨著發展，AI將來能夠提供給我們更多的協助，若能妥善利用AI確實會有很多幫助。

**Comment by Lai27330** [comment]
能利用AI來幫助處理繁瑣重複的真的事非常方便呢，做事效率也會提高。

**Comment by awkward-willy** [comment]
讓科技輔佐人們的生活會是未來的一大趨勢
不過同時也會出現科技取代人力等的問題
